scripts/log2dch

```diff
@@ -67,11 +70,13 @@ debug() {
 bzr_log_to_dch() {
     local line="" commit="" bugs="" bug=""
     local func="${1:-print_dch}"
+    local skip_bugs="${2:-false}"
```

Comment:
```suggestion
    local skip_bugs="${2:-"false"}"
```

Reply:
but this actually is wierd.  because you have the inner and outer quotes.
Chad's is more readable.  I honestly do not know how this one gets parsed.

Reply:
I could use single ticks in the inner; 

The boolean true/false is certainly more compact.  Good enough.